README.md

```diff
@@ -10,45 +10,45 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 
 | Angielski            | Polskawy            | Polski                                |
 | :------------------- | :------------------ | :------------------------------------ |
-| approve              | zapruwować          | zaakceptować                          |
+| approve              | zaapruwować         | zaakceptować                          |
 | authentication       | autentykacja        | uwierzytelnienie (!)                  |
 | authorization        |                     | autoryzacja                           |
 | build                | bild                | budowanie                             |
-| button               | baton               | przycisk                              |
-| cancel               | kanselować          | anulować                              |
+| button               | baton               | przycisk, guzik                       |
+| cancel               | kanselować, anulacja| anulować                              |
 | case                 | kejs                | przypadek                             |
 | confirm              | konfirmować         | potwierdzić                           |
-| content              |                     | zawartość, treść                      |
+| content              | kontent             | zawartość, treść                      |
 | custom               | kastomowy           | specjalny, nietypowy, niestandardowy  |
 | default              | difoltowo           | domyślnie                             |
 | deploy               | zdiplojować         | wdrożyć                               |
-| edge case            |                     | przypadek brzegowy                    |
-| event                |                     | zdarzenie                             |
-| feature              | ficzer              | funkcja                               |
-| feedback             |                     | informacja zwrotna                    |
+| edge case            | edż kejs            | przypadek brzegowy                    |
+| event                | iwent               | zdarzenie, wydarzenie                 |
```

Comment:
:+1:

Reply:
W myśl https://github.com/nurkiewicz/polski-w-it/pull/23 chciałbym unikać wersji "polskawej", która jest tylko zapisem fonetycznym. Natomiast same tłumaczenia jak najbardziej spoko 👍